.github/workflows/release-binary-assets.yml

```diff
@@ -55,6 +55,18 @@ jobs:
           command: run
           use-cross: ${{ matrix.cross }}
           args: --features vendored-openssl --release --target=${{ matrix.target }} -- --version
+      - name: Install cargo-deb and create a deb package
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          cargo install cargo-deb
+          cargo deb
```

Comment:
could you please figure out if there is a good way how to pass over the args 

```
  --features vendored-openssl --release --target=${{ matrix.target }} -- --version
```

so that we make sure the generated deb file does use the `vendored-openssl` feautre and has a matching target architecture, otherwise we would generate here multiple deb packages for linux. see the matrix configuration above.

Reply:
@sassman Sorry I wasnt available is this still an issue? I can see that the PR is merged

Reply:
@John15321 I believe it is an issue and the PR is still open :)

Reply:
> @John15321 I believe it is an issue and the PR is still open :)



> @John15321 I believe it is an issue and the PR is still open :)

Oh sorry, maybe its better to not reply just after waking up lol